proto/interchain_security/ccv/provider/v1/tx.proto

```diff
@@ -23,34 +23,45 @@ service Msg {
   rpc AssignConsumerKey(MsgAssignConsumerKey) returns (MsgAssignConsumerKeyResponse);
   rpc SubmitConsumerMisbehaviour(MsgSubmitConsumerMisbehaviour) returns (MsgSubmitConsumerMisbehaviourResponse);
   rpc SubmitConsumerDoubleVoting(MsgSubmitConsumerDoubleVoting) returns (MsgSubmitConsumerDoubleVotingResponse);
-  rpc ConsumerAddition(MsgConsumerAddition) returns (MsgConsumerAdditionResponse);
-  rpc ConsumerRemoval(MsgConsumerRemoval) returns (MsgConsumerRemovalResponse);
+  rpc ConsumerAddition(MsgConsumerAddition) returns (MsgConsumerAdditionResponse) {
+    option deprecated = true;
+  };
+  rpc ConsumerRemoval(MsgConsumerRemoval) returns (MsgConsumerRemovalResponse) {
+    option deprecated = true;
+  };
+  rpc CreateConsumer(MsgCreateConsumer) returns (MsgCreateConsumerResponse);
+  rpc UpdateConsumer(MsgUpdateConsumer) returns (MsgUpdateConsumerResponse);
+  rpc RemoveConsumer(MsgRemoveConsumer) returns (MsgRemoveConsumerResponse);
   rpc UpdateParams(MsgUpdateParams) returns (MsgUpdateParamsResponse);
   rpc OptIn(MsgOptIn) returns (MsgOptInResponse);
   rpc OptOut(MsgOptOut) returns (MsgOptOutResponse);
   rpc SetConsumerCommissionRate(MsgSetConsumerCommissionRate) returns (MsgSetConsumerCommissionRateResponse);
-  rpc ConsumerModification(MsgConsumerModification) returns (MsgConsumerModificationResponse);
+  rpc ConsumerModification(MsgConsumerModification) returns (MsgConsumerModificationResponse) {
+    option deprecated = true;
+  }
   rpc ChangeRewardDenoms(MsgChangeRewardDenoms) returns (MsgChangeRewardDenomsResponse);
 }
 
 
 message MsgAssignConsumerKey {
-  option (cosmos.msg.v1.signer) = "signer";
+  option (cosmos.msg.v1.signer) = "submitter";
   option (gogoproto.equal) = false;
   option (gogoproto.goproto_getters) = false;
 
 
-  // The chain id of the consumer chain to assign a consensus public key to
-  string chain_id = 1;
+  // [DEPRECATED] use `consumer_id` instead
+  string chain_id = 1 [deprecated = true];
   // The validator address on the provider
   string provider_addr = 2 [ (gogoproto.moretags) = "yaml:\"address\"" ];
   // The consensus public key to use on the consumer.
   // in json string format corresponding to proto-any, ex:
   // `{"@type":"/cosmos.crypto.ed25519.PubKey","key":"Ui5Gf1+mtWUdH8u3xlmzdKID+F3PK0sfXZ73GZ6q6is="}`
   string consumer_key = 3;
 
-  // Tx signer address
-  string signer = 4 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  string submitter = 4 [(cosmos_proto.scalar) = "cosmos.AddressString"];
```

Comment:
Revert the changes to not break compatibility. If you do this you will not be able to unmarshall historic Txs.